tools/combine_metaphlan_humann/combine_metaphlan_humann.xml

```diff
@@ -0,0 +1,112 @@
+<tool id="combine_metaphlan_humann" name="Combine MetaPhlAn and HUMAnN outputs" version="0.3.0" profile="22.01">
+    <description>to relate genus/species abundances and gene families/pathways abundances</description>
+    <requirements>
```

Comment:
no requirements? not even python?

Reply:
It requires python to run, but the prev. version was OK with the python of the environment. Should I still add?

Reply:
Anyways, done.